data/db/mil-sigma.xml

```diff
@@ -224,4 +224,15 @@
         </calibration>
     </lens>
 
+    <lens>
+        <maker>Sigma</maker>
+        <model>Sigma 16mm f/1.4 DC DN | Contemporary C 017</model>
+        <mount>Sony E</mount>
```

Comment:
Is this really the only mount?

Reply:
The lense is also available for Micro 4/3, but that one is already covered by https://github.com/lensfun/lensfun/commit/dc7557d5c8f5bc62e572ef1d462efdfcf87a5107 and has different calibration data as far as I can judge, also the results doesn't look good for my photos when choosing the already avalable Micro 4/3 entry manually. I'm new to this stuff so I might be missing how to properly combine data within the xml.

Reply:
Ah, good to know. We do the following in such cases: Put the two datasets next to each other, add all mounts to both entries. In a future Lensfun version, we hope to be able to merge such duplicates.

Reply:
> Put the two datasets next to each other

check
> add all mounts to both entries

did that for the new entry, not sure if adding emount to the already existing should be part of this PR. I could do a follow-up PR or add the emount to the existing entry with a second commit in this PR. What do you prefer?

Reply:
You’re right, I do that separately.